src/river/river.py

```
@@ -67,240 +71,308 @@ def transform_bbox(x_min, x_max, y_min, y_max, from_epsg, to_epsg):
+            return
+
+        # Загрузка и подготовка DEM
+        if not progress.update(10, "Загрузка цифровой модели рельефа"):
+            return
+
+        # Скачивание DEM
+        dem_path = download_dem(bbox, project_folder)
+
+        # Создание перепроецированного DEM в EPSG:3857
+        dem_3857 = f"{project_folder}/river_dem_3857.tif"
+        gdal.Warp(
```

Ruff (0.11.9)

Undefined name `gdal`

(F821)